src/Symfony/Component/Scheduler/RecurringMessage.php

```diff
@@ -42,30 +45,40 @@ private function __construct(
      */
     public static function every(string|int|\DateInterval $frequency, object $message, string|\DateTimeImmutable $from = new \DateTimeImmutable(), string|\DateTimeImmutable $until = new \DateTimeImmutable('3000-01-01')): self
```

Comment:
Would it make sense to type hint `MessageProviderInterface|object` or maybe add a `@param` doc that explains the two possibilities? Or maybe both :)

Reply:
That typehint triggers this error for me:
```
PHP Fatal error:  Type Symfony\Component\Scheduler\Trigger\MessageProviderInterface|object contains both object and a class type, which is redundant in \symfony\src\Symfony\Component\Scheduler\RecurringMessage.php on line 64
```

I did add it to the doc of `RecurringMessage::trigger()`

Reply:
ok, let's only add phpdocs then.

Reply:
Does doc for just `::trigger()` suffice or would it also be good on `every()` and `cron()` ?

Reply:
I think we should do it eveywhere it's supported to make it clear.